pkg/node/Dockerfile.build

```diff
@@ -5,4 +5,14 @@ ADD     health_probe    health_probe
 # Remove bash packet to get rid of related CVEs
 RUN     apt update --no-install-recommends -y -q; apt remove --no-install-recommends -y --allow-remove-essential -q bash; apt install --no-install-recommends -y -q util-linux parted xfsprogs lvm2 gdisk strace udev net-tools
 
-
+# Don't rely on udevd, it isn't available in some combinations of host/guest kernels.
```

Comment:
`isn't available` not precise term here. `not compatible` due to poll/epoll APIs sounds better. Please add md doc with your finding and post refence here

Reply:
Added docs, removed changes to Dockerfile due to after repeated tests the issue with lvm was not reproduced